### PR TITLE
UI: Added localization changes part 2

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Added localization changes part 2

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@
  *  limitations under the License.
  */
 
+import i18next from 'i18next';
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx:14]: @ShaileshParmar11, I recommend directly importing the `t` from `i18next` here.

File: openmetadata-ui/src/main/resources/ui/src/components/SampleDataTopic/SampleDataTopic.tsx [hunk 2]
```diff
@@ -13,8 +13,10 @@
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { Typography } from 'antd';
+import { t } from 'i18next';
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/SampleDataTopic/SampleDataTopic.tsx:16]: @ShaileshParmar11 , Can you check if we are using this, I can see you have imported `useTranslation` as well.

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/SampleDataTopic/SampleDataTopic.tsx (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/SampleDataTopic/SampleDataTopic.tsx:16]: I will have a look thanks!

**Comment by ShaileshParmar11** [comment]
I address the above comments in my next PR of localization @Sachin-chaurasiya

**Pull request merged**